website/static/js/filepage/index.js

```diff
@@ -198,12 +199,17 @@ var FileViewPage = {
         $('.file-view-panels').removeClass().addClass('file-view-panels').addClass(fileViewPanelsLayout);
 
         m.render(document.getElementById('toggleBar'), m('.btn-toolbar[style=margin-top:20px]', [
-            ctrl.canEdit() ? m('.btn-group', {style: 'margin-left: 0;'}, [
-                m('.btn.btn-sm.btn-danger.file-delete', {onclick: $(document).trigger.bind($(document), 'fileviewpage:delete')}, 'Delete')
+            ctrl.viewUrl ? m('.btn-group', {}, [
+                m('a', {href: ctrl.viewUrl, target: '_blank', class: 'btn btn-sm btn-success'}, 'View ', [
```

Comment:
@GaryKriebel The problem is that there are now 2 "View" buttons that do different things.

I suggested changing to "View on Dropbox" but that makes the button large (which may or may not be a problem).

@caseyrollins Also, I just checked other parts of the site, and it appears that the icon should go *before* the text in the button.